modules/machine-health-checks-resource.adoc

```diff
@@ -36,7 +36,7 @@ spec:
 <2> Specify a label for the machine pool that you want to check.
 <3> Specify the machine set to track in `<cluster_name>-<label>-<zone>` format. For example, `prod-node-us-east-1a`.
 <4> Specify the timeout duration for a node condition. If a condition is met for the duration of the timeout, the machine will be remediated. Long timeouts can result in long periods of downtime for a workload on an unhealthy machine.
-<5> Specify the amount of unhealthy machines allowed in the targeted pool. This can be set as a percentage or an integer.
+<5> Specify the amount of remediating machines allowed in the targeted pool. This can be set as a percentage or an integer. maxUnhealthy prevents further remediation if the cluster is already partially unhealthy.
```

Comment:
Coming back to this, I think `remediating`, while technically more accurate, isn't really the way a user would understand this. I think the bit at the end makes sense but we should probably switch remediating back to unhealthy.
```suggestion
<5> Specify the number of unhealthy machines allowed in the targeted pool. This can be set as a percentage or an integer. maxUnhealthy prevents further remediation if the cluster is already partially unhealthy.
```
If you think this is unclear still, perhaps we can restructure the sentence to make it more obvious?
```
Specify a limit for the number of unhealthy machines allowed in a target pool for safe operation of the machine health check. If this limit is exceeded, further remediation is blocked to allow manual intervention to resolve wider issues within the cluster.
```

Reply:
I cringed at the "amount" word myself, but since we also support percentage (e.g. 40%) it may be fine. As it is unrelated to the purpose of this PR I'd prefer to leave it out.

I think that the change from unhealthy to remediating is crucial. We need to educate the reader that MHC cannot control whether their host is healthy or not. This is a fact of physical reality (e.g somebody turned off the TOR switch). MHC can only choose what to do with this fact. To me `maxUnhealthy` is a legacy term that should be explained away, and this is what I try to do in this PR.

Reply:
Does the second part of my comment improve it do you think? Trying to express that this is a limit for safe operation

Reply:
I find "number of unhealthy machines allowed" most confusing. As if it is the number of nodes we would tolerate as unhealthy, and start working only if things gets worse. It through me off without repair.

Maybe you can explain more why a user would misunderstand what you consider the more accurate term?

Reply:
I also had this same confusion... "number of unhealthy machines allowed" means, at least to me, how many nodes we'll tolerate being `NotReady` before we take action, which as we know now is not as intended.

Reply:
I appreciate the confusion, hence my wanting to add the term limit into the sentence.

I suggest not to use the term `remediating machines` because we don't use that term anywhere else in the docs. We say that the machine goes unhealthy and then we remediate it. The unhealthy part belongs to the machine, not the remediating. So there's no such thing as a `remediating machine`, there is however a `machine that is being remediated`. It's a subtle but important distinction IMO

Reply:
so how about
```suggestion
<5> Specify the amount of machines allowed to be concurrently remediated in the targeted pool. This can be set as a percentage or an integer. maxUnhealthy prevents further remediation if the cluster is already partially unhealthy.
```

Reply:
That last suggestion works for me